inc/itilsolution.class.php

```diff
@@ -325,17 +329,21 @@ function post_addItem() {
       }
 
       $item = $this->item;
-      $status = $item::CLOSED;
-      $autoclosedelay =  Entity::getUsedConfig(
-         'autoclose_delay',
-         $this->item->getEntityID(),
-         '',
-         Entity::CONFIG_NEVER
-      );
+      $status = $item::SOLVED;
+
+      //handle autoclose, for tickets only
+      if ($item->getType() == Ticket::getType()) {
+         $autoclosedelay =  Entity::getUsedConfig(
+            'autoclose_delay',
+            $this->item->getEntityID(),
+            '',
+            Entity::CONFIG_NEVER
+         );
 
          // 0 = immediatly
-      if ($autoclosedelay != 0) {
-         $status = $item::SOLVED;
+         if ($autoclosedelay != 0) {
```

Comment:
` == 0` -> `CLOSED`, like in https://github.com/glpi-project/glpi/pull/4830/files#diff-c1fd7f23e92e1731809fc183d9edc997R352 ?

Reply:
Indeed, I've fixed